engine/src/main/java/com/redhat/cloud/notifications/templates/Edge.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Edge" to read templates from resources/templates/Edge
+public class Edge implements EmailTemplate {
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals("image-creation")) {
+            return Templates.imageCreationTitle();
+        }
+        return Templates.updateDeviceTitle();
+    }
+
+    @Override
+    public TemplateInstance getBody(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals("image-creation")) {
+            return Templates.imageCreationBody();
+        }
+        return Templates.updateDeviceBody();
+    }
+
+    @Override
+    public boolean isSupported(String eventType, EmailSubscriptionType type) {
+        return true;
+    }
+
+    @Override
+    public boolean isEmailSubscriptionSupported(EmailSubscriptionType type) {
+        return true;
```

Comment:
It looks like both templates from this PR are for `instant` notifications (the other type of notification is the `daily` aggregation). If that's correct, this line should be changed to:
```suggestion
        return type == EmailSubscriptionType.INSTANT;
```